server/conf/helper/feature-flags.conf

```diff
@@ -55,10 +55,6 @@ no_duplicate_questions_for_migration_enabled = ${?NO_DUPLICATE_QUESTIONS_FOR_MIG
 show_not_production_banner_enabled = false
 show_not_production_banner_enabled = ${?SHOW_NOT_PRODUCTION_BANNER_ENABLED}
 
-# Fast forward feature to keep draft applications on latest program
-fastforward_enabled = false
```

Comment:
comment (non-blocking): I feel fine removing this flag as is now, but my understanding is that generally we want to turn the flag on by default for awhile before removing.

Reply:
Yeah, I think I've let myself get less stringent when removing ones that have been around and in use for a long time. If you have concerns on this one I can back this out.

Reply:
Sounds good. Appreciate you raising the issue.